added main.js and style.css

Pitclair opened the pull request:
functions in main.js have doc comments
style.css is mandatory for main.js functions to work
a list of necessary requirements for running the js function is added to the main.js

Pitclair committed:
added main.js and style.css

functions in main.js have doc comments
style.css is mandatory for main.js functions to work
a list of necessary requirements for running the js function is added to the main.js

farzad1132 commented on main.js:
```diff
@@ -0,0 +1,219 @@
+
+    <script src="main.js"></script>
+	<script src="https://code.jquery.com/jquery-3.4.1.min.js"
+	    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```
these includes exist in the map.html

farzad1132 commented on main.js:
```diff
@@ -0,0 +1,219 @@
+    tileSize: 512,
+    zoomOffset: -1,
+    accessToken: 'pk.eyJ1IjoicGl0Y2xhaXIiLCJhIjoiY2s3MDZmZGcwMDB1NDNtcDBmZ3pjdWp2NyJ9.MC35oYwFqBb-Zndvv2yiHQ'
+}).addTo(mymap);
```
also these lines

farzad1132 commented:
also, point where I have to pass the link click event?

farzad1132 committed:
skype review edits

farzad1132 committed:
map.html header updated

Pitclair commented:
updated version - with only the main functions
[updated-js-and-css.zip](https://github.com/farzad1132/NetPlanner/files/4333303/updated-js-and-css.zip)

farzad1132 committed:
Wavelength popup added

farzad1132 committed:
legend added to main map

Pitclair committed:
removed wavelength number for gray lines

handleMouseOverLines function added as a callback function to drawLines function ( no more need to call the handleMouseOverLines  as a global function)
a comparing of commits will show the changes.
please also replace the body on the functions with the ones provided here

farzad1132 committed:
lambda view completed [ add overall results ]
